Description: Show GNOME Documents in menus everywhere
Author: Jeremy Bicha <jbicha@ubuntu.com>
Index: gnome-boxes-3.8.4/data/gnome-boxes.desktop.in
===================================================================
--- gnome-boxes-3.8.4.orig/data/gnome-boxes.desktop.in	2013-08-27 19:47:39.924379732 +0200
+++ gnome-boxes-3.8.4/data/gnome-boxes.desktop.in	2013-08-27 19:47:39.924379732 +0200
@@ -8,6 +8,5 @@
 Terminal=false
 Type=Application
 StartupNotify=true
-OnlyShowIn=GNOME;
 Categories=GNOME;GTK;System;
 MimeType=x-content/bootable-media;
Index: gnome-boxes-3.8.4/data/gnome-boxes.desktop.in.in
===================================================================
--- gnome-boxes-3.8.4.orig/data/gnome-boxes.desktop.in.in	2013-08-27 19:47:39.924379732 +0200
+++ gnome-boxes-3.8.4/data/gnome-boxes.desktop.in.in	2013-08-27 19:47:39.924379732 +0200
@@ -8,6 +8,5 @@
 Terminal=false
 Type=Application
 StartupNotify=true
-OnlyShowIn=GNOME;
 Categories=GNOME;GTK;System;
 MimeType=x-content/bootable-media;
