Description: Show GNOME Documents in menus everywhere
Author: Jeremy Bicha <jbicha@ubuntu.com>
--- a/data/gnome-boxes.desktop.in
+++ b/data/gnome-boxes.desktop.in
@@ -8,5 +8,4 @@
 Terminal=false
 Type=Application
 StartupNotify=true
-OnlyShowIn=GNOME;
 Categories=GNOME;GTK;System;
--- a/data/gnome-boxes.desktop.in.in
+++ b/data/gnome-boxes.desktop.in.in
@@ -8,5 +8,4 @@
 Terminal=false
 Type=Application
 StartupNotify=true
-OnlyShowIn=GNOME;
 Categories=GNOME;GTK;System;
